WordPress/src/test/java/org/wordpress/android/ui/prefs/categories/CategoriesListViewModelTest.kt

```diff
@@ -113,6 +113,57 @@ class CategoriesListViewModelTest : BaseUnitTest() {
         assertTrue(uiStates.last() is GenericError)
     }
 
+    @Test
+    fun `given error occurs, when retry is invoked, then loading is displayed`() {
+        whenever(networkUtilsWrapper.isNetworkAvailable())
+                .thenReturn(true)
```

Comment:
Nitpick: `thenReturn` here and in the other tests can be moved to the preceding line to fit it in one line.

Reply:
@ashiagr Nice, I think this will be a good improvement. Done in https://github.com/wordpress-mobile/WordPress-Android/pull/15603/commits/0c0b0970c2d553f48f7f4f3e4076761945ccacd1